### PR TITLE
Added Past Orders to Profile Page

### PR DESCRIPTION
**Pull request by EthanAuyeung**
[//]: # "These comments are meant for your reference. They are invisible and don't need to be deleted!"

### Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."
Added the order history section to the profile page, includes 3 most recent item photos, status of the order, order number, and date it was submitted

### Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy!"
Screenshot 2024-03-19 at 3 19 35 PM

## How to review
[//]: # "Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?"
Checkout branch and run npm dev

## Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### Online sources
[//]: # "Optional - copy links to any tutorials or documentation that was useful to you when working on this PR"

### Related PRs
[//]: # "Optional - related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

[//]: # "This tags the project leader as a default. Feel free to change, or add on anyone who you should be in on the conversation."
🛍 CC: @EthanAuyeung

### CODE REVIEW

File: src/app/profileScreen/page.tsx [hunk 1]
```diff
@@ -84,17 +96,111 @@ function FavoriteSection(props: {
...
+    return (
+      <main>
+        <OrderHistory>
+          <HeaderDiv>
+            <Heading2>Order History</Heading2>
+            <ViewAllButton destination="./orderPage" />
+          </HeaderDiv>
+          <div
+            style={{
+              marginTop: '20px',
```
Comment (BuyankhuuTsCAl) [inline src/app/profileScreen/page.tsx:176]: it might be cleaner to not have inline styling

File: src/app/profileScreen/page.tsx [hunk 2]
```diff
@@ -84,17 +96,111 @@ function FavoriteSection(props: {
...
+              }}
+            >
+              {Orders[0].status}
+            </Body2Bold>
+            <MostRecentOrder>
+              {firstOrderProducts.map(product => (
+                <div
+                  key={product.id}
+                  style={{
+                    width: '85px',
```
Comment (BuyankhuuTsCAl) [inline src/app/profileScreen/page.tsx]: same idea but don't use grey and use the color class

**Review by BuyankhuuTsCAl — COMMENTED** [review]
Just wanted to give feedback before I go AWOL
